cockpit/src/main/java/com/polidea/cockpit/view/BooleanParamView.kt

```diff
@@ -22,8 +22,9 @@ class BooleanParamView(context: Context, attrs: AttributeSet? = null, defStyleAt
     }
 
     init {
-        LayoutInflater.from(context).inflate(R.layout.cockpit_boolean_param_line, this,true)
+        LayoutInflater.from(context).inflate(R.layout.cockpit_boolean_param_line, this, true)
         cockpit_boolean_param_value.isChecked = value
         cockpit_boolean_param_name.text = paramName
+        cockpit_boolean_param_name.isSelected = true
```

Comment:
I suppose, you select this view to make text view autoscrolling (horizontally) when text is too long. I wonder if it's needed. It looks a little weird when you switch focus between edit texts: autoscroll starts from beginning on each focus change.